CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing
+
+## Basic Steps
+
+If you would like to contribute enhancements or fixes, please do the following:
+
+1.  Fork the plugin repository
+2.  Run `npm install` to setup all dependencies
+3.  Hack on a separate topic branch created from the latest `master`
+4.  Commit and push the topic branch
+5.  Make a pull request
+6.  Welcome to the club!
+
+## Guidelines
+
+Please note that modifications should follow these coding guidelines:
+
+*   Indent is 2 spaces
+*   Code should pass the `eslint` linter
+*   Vertical whitespace helps readability, don’t be afraid to use it
+
+## Building
+
+The simplest procedure is to just run `npm run watch` in a terminal. This will
+launch `ucompiler` and tell it to watch the `src` directory, automatically
+recompiling any file that is updated for you. If you prefer to run this step
+manually, running `npm install` will also run `ucompiler go` for you.
```

Comment:
It does handle that, but it still has the scan other files to validate their end-file does not match a known glob. I have the enhanement in mind where it'll be able to ignore parent dirs of non-used files